entrypoint.sh

```diff
@@ -3,7 +3,7 @@ printenv | grep -E 'PATH|S3_BUCKET_NAME|S3_ACCESS_KEY|S3_SECRET_KEY|S3_REGION|MY
 sed -i -e "s/CRON_EXPR/${CRON_EXPR}/g" /etc/cron.d/backup-cron
 
 ## Start metrics endpoint
-while true; do { printf 'HTTP/1.0 200 OK\r\nContent-Length: %d\r\n\r\n' "$(wc -c < /root/metrics.txt)"; cat /root/metrics.txt; } | nc -l -p 9300; done &
+while true; do { printf 'HTTP/1.0 200 OK\r\nContent-Length: %d\r\n\r\n' "$(wc -c < /root/metrics.txt)"; cat /root/metrics.txt; } | nc -l -p 9300 >> netcat.log; done &
```

Comment:
In welchem Verzeichnis liegt die netcat.log? Kannst du das ansonsten in das gleiche Verzeichnis machen, in der auch die anderen Log-Dateien liegen?